packages/api/src/graphql/query.public.ts

```diff
@@ -426,6 +427,14 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
         const updatedInvoices = await dbAdapter.invoice.getInvoicesByUserID(user.id)
         return updatedInvoices.find(invoice => invoice !== null && invoice.id === id)
       }
+    },
+    getSystemInformation: {
```

Comment:
This information should never be public. Avoid telling the world, which bugs you have in your application.